server/portal/apps/users/utils.py

```diff
@@ -262,7 +262,11 @@ def remove_user(project_id, user_id):
 
 
 def check_user_groups(username, groups):
-    return any(
-        user['username'] == str(username)
-        for group in groups for user in get_project_users_from_name(group)
-    )
+    try:
+        return any(
+            user['username'] == str(username)
+            for group in groups for user in get_project_users_from_name(group)
+        )
+    except:
```

Comment:
Can we do this to also log exception. It will help us debug the issue  - also fixes lint error

```
    except Exception as e:
        logger.error(f"Issue checking user groups for user: {username}. Error: {e}")
        return False
```

Reply:
This is just FYI: Another implementation option is to catch and log in [launch_setup_checks in the view](https://github.com/TACC/Core-Portal/blob/main/server/portal/apps/auth/views.py#L80-L92). 

The above implementation works too. TAS could be temporarily down and dealing with it at each call might give us some benefits.